.travis.yml

```diff
@@ -12,13 +12,20 @@ env:
 before_install:
 - docker pull exaile/exaile-testimg:${IMGTAG}
 
+install:
+- pip install --user pytest-flakes
```

Comment:
This is happening on the travis VM, not in the docker images. That's most likely why it's failing.